hack/build-cross.sh

```diff
@@ -29,7 +29,8 @@ fi
 
 # Build the primary client/server for all platforms
 OS_BUILD_PLATFORMS=("${platforms[@]}")
-os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
+# Create a sub-shell so that we don't pollute the outer environment
+( os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}" )
```

Comment:
Why not make build_binaries itself run its work in a subshell, rather than propagating up and out?